src/views/HomePage.vue

```diff
@@ -1,18 +1,18 @@
 <template>
-  <div class="home">
+  <div class="home-page">
     <img alt="Vue logo" src="../assets/logo.png" />
-    <HelloWorld msg="Welcome to Your Vue.js App" />
+    <Header msg="Welcome to Your Vue.js App" />
```

Comment:
Please move the `Header` to the `App` component, in case we have other pages, the `Header` and the `Footer` should keep the same, only the page content will change.